src/neuroconv/datainterfaces/icephys/baseicephysinterface.py

```diff
@@ -33,6 +26,17 @@ def get_source_schema(cls) -> dict:
         return source_schema
 
     def __init__(self, file_paths: list):
+        # Check if the ndx_dandi_icephys module is available
+        dandi_icephys_spec = importlib.util.find_spec("ndx_dandi_icephys")
+        if dandi_icephys_spec is not None:
+            dandi_icephys_module = importlib.util.module_from_spec(dandi_icephys_spec)
+            dandi_icephys_spec.loader.exec_module(dandi_icephys_module)
+            self.DandiIcephysMetadata = getattr(dandi_icephys_module, "DandiIcephysMetadata")
+            self.HAVE_NDX_DANDI_ICEPHYS = True
+        else:
+            self.DandiIcephysMetadata = None
+            self.HAVE_NDX_DANDI_ICEPHYS = False
```

Comment:
Curious what you would think about setting these as the default at the class level (next to `ExtractorModuleName`)?

Reply:
I don't like it really. They are determined at import times, how we could make attributes of them?